Dockerfile

```diff
@@ -78,7 +78,7 @@ RUN chown -R codegate /var/lib/nginx && \
     chown -R codegate /var/log/nginx && \
     chown -R codegate /run
 
-COPY nginx.conf /etc/nginx/conf.d/default.conf
+COPY nginx.conf /etc/nginx/nginx.conf
```

Comment:
is there any system-default nginx.conf? Which of them takes precedence? Just making sure we won't block ourselves from receiving changes to defaults in the future

Reply:
right, that's exactly what I'm overwriting here. We no longer take into account the default nginx.conf and instead write our own.